src/test/suite/scanner.test.ts

```diff
@@ -42,14 +44,11 @@ suite('ScanRunner', () => {
 
                 // ===== ASSERTIONS =====
                 // Verify that the right arguments were created.
-                expect(args).to.have.lengthOf(7, 'Wrong number of args');
-                expect(args[0]).to.equal('scanner', 'Wrong arg');
-				expect(args[1]).to.equal('run', 'Wrong arg');
-                expect(args[2]).to.equal('--target', 'Wrong arg');
-                expect(args[3]).to.equal(targets.join(','), 'Wrong arg');
-                expect(args[4]).to.equal('--engine', 'Wrong arg');
-                expect(args[5]).to.equal('pmd,retire-js', 'Wrong arg');
-                expect(args[6]).to.equal('--json', 'Wrong arg');
+                // expect(args).to.have.lengthOf(3, 'Wrong number of args');
```

Comment:
Did you mean to have commented out code here? Maybe just remove this line if not needed.

Reply:
Done